response_operations_ui/templates/close-conversation.html

```diff
@@ -4,7 +4,7 @@
   <div role="main" id="main" class="page__main">
     <div class="spoke__header">
       <div class="spoke__header-title">
-        <h1 class="jupiter">Close conversation</h1>
+        <h1 class="u-fs-xl">Close conversation</h1>
```

Comment:
The title now is a lot smaller using this

Reply:
The spoke.html is still using the old cdn hash so it needs to be updated to 2.0.0 to fix it.

Reply:
ah, that'll probably fix a bunch of these issues

Reply:
Yeah, there are two locations (because why wouldn't there be) to change the hash